ui/src/app/utils/stores.tsx

```diff
@@ -73,11 +73,11 @@ export const abortRuntimeOperationForWorkspace = (workspaceNamespace: string) =>
 
 // runtime store states: undefined(initial state) -> Runtime (user selected) <--> null (delete only - no recreate)
 interface RuntimeStore {
-  workspaceNamespace: string;
+  workspaceNamespace: string | null | undefined;
   runtime: Runtime | null | undefined;
 }
 
-export const runtimeStore = atom<RuntimeStore>(undefined);
+export const runtimeStore = atom<RuntimeStore>({workspaceNamespace: undefined, runtime: { dataprocConfig: {}}});
```

Comment:
I'm not sure why runtime would start as being defined with a stub value, and in particular why it would be set to dataprocConfig. There should specifically be *no* runtime when we are not in a workspace context, which is the initial state. Was this for testing purposes? If so, I'd direct you here: https://github.com/all-of-us/workbench/blob/f272563b7256f6e10d9acc537b7f605f65dee368/ui/src/test.ts#L30

Reply:
I was running into some typing issues earlier. I updated this to be undefined.